flink-table/flink-sql-client/src/test/resources/sql/table.q

```diff
@@ -926,6 +927,52 @@ desc orders3;
+|   ptime | TIMESTAMP_LTZ(3) *PROCTIME* | FALSE |           | AS PROCTIME() |                            |  notice: computed column |
++---------+-----------------------------+-------+-----------+---------------+----------------------------+--------------------------+
+5 rows in set
+!ok
+
+show columns in orders3 like 'p%';
++---------+-----------------------------+-------+-----+---------------+-----------+-------------------------+
+|    name |                        type |  null | key |        extras | watermark |                 comment |
++---------+-----------------------------+-------+-----+---------------+-----------+-------------------------+
+| product |                 VARCHAR(32) |  TRUE |     |               |           |                         |
+|   ptime | TIMESTAMP_LTZ(3) *PROCTIME* | FALSE |     | AS PROCTIME() |           | notice: computed column |
++---------+-----------------------------+-------+-----+---------------+-----------+-------------------------+
+2 rows in set
+!ok
+
+# test show create table
+show create table orders3;
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                                                                                                                                                                                                                                                                                                                                                                                                                                         result |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| CREATE TABLE `default_catalog`.`default_database`.`orders3` (
+  `user` BIGINT NOT NULL COMMENT 'this is the first column',
+  `product` VARCHAR(32),
+  `amount` INT,
+  `ts` TIMESTAMP(3) COMMENT 'notice: watermark',
+  `ptime` AS PROCTIME() COMMENT 'notice: computed column',
+  WATERMARK FOR `ts` AS `ts` - INTERVAL '1' SECOND,
+  CONSTRAINT `PK_3599338` PRIMARY KEY (`user`) NOT ENFORCED
```

Comment:
Please also add a single quote in the comment to verify the escaping.

Reply:
done